aws/templates/name/name_cognito.ftl

```diff
@@ -1,24 +1,28 @@
 [#-- Cognito User Pool --]
 
-[#function formatUserPoolName tier component ]
+[#function formatUserPoolName occurrence extensions...]
```

Comment:
Like ids, these could simply to 

these could simplify further to

these could simplify further to

[#return formatSegmentFullName(occurrence.Core.Name, extensions) ]

the formatName is redundant even in the existing form